scripts/alert_emails/raw_html.html

```diff
@@ -0,0 +1,1006 @@
+<!DOCTYPE html
```

Comment:
wow wow wow...  You weren't kidding about this being a huge blob of html. 😁 

Is this all mostly generated via Campaign Monitor, I guess? Like, a lot of it could maybe theoretically be removed?  Or we could try to start from scratch and come up with something simpler?

It's probably fine for now.

At minimum though, we probably need like a comment explaining what this file is and how it's used (you manually edit it and then import it into the right place in Campaign Monitor)?

Reply:
when you ignore the css it's mostly pretty readable!